API/Import.php

```diff
@@ -119,17 +156,10 @@ public function importContainerVersion($exportedContainerVersion, $idSite, $idCo
     {
         $this->checkImportContainerIsPossible($exportedContainerVersion, $idSite, $idContainer);
 
-        foreach ($this->tags->getContainerTags($idSite, $idContainerVersion) as $tag) {
```

Comment:
@tsteur any specific reason we deleted 1 tag within a container at a time instead of deleting all at once like in this PR ?